### PR TITLE
[activity] Use short IDs when referencing issues

### PR DESCRIPTION
**Pull request by dcramer**
screenshot 2017-04-22 11 21 09

### CODE REVIEW

**Comment by dcramer** [comment]
This pulls in the activity feed changes (barring references) that I made during hack week

File: src/sentry/static/sentry/app/components/activity/item.jsx [hunk 1]
```diff
@@ -51,133 +51,129 @@ const ActivityItem = React.createClass({
...
+      : null;
 
     switch (item.type) {
       case 'note':
-        return tct('[author] commented on [link:an issue]', {
+        return tct('[author] commented on [issue]', {
           author: author,
-          link: (
-            <Link
-              to={`/${orgId}/${project.slug}/issues/${issue.id}/activity/#event_${item.id}`}
```
Comment (macqueen) [inline src/sentry/static/sentry/app/components/activity/item.jsx]: looks like you're actually changing the link here--not sure if that was intentional?

**Review by macqueen — APPROVED** [review]

File: src/sentry/static/sentry/app/components/activity/item.jsx (hunk 1 above)
Comment (dcramer) [inline src/sentry/static/sentry/app/components/activity/item.jsx]: oops, nope

**Comment by benvinegar** [comment]
Awesome.

**Pull request merged**